src/atoms/FeaturedSection/index.js

```diff
@@ -0,0 +1,37 @@
+import classNames from "classnames"
+import PropTypes from "prop-types"
+import React, { Component } from "react"
+import styles from "./featuredSection.module.css"
+import { Link } from "gatsby"
+
+class FeaturedSection extends Component {
+  render() {
+    let sectionClass = classNames(
+      styles[`featuredSection`],
+      styles[`featuredSectionLevel${this.props.level}`]
+    )
+
+    return (
+      <section className={sectionClass}>
+        <h3>{this.props.title}</h3>
+        {this.props.children}{" "}
+        <Link to="/" className={styles.callLink}>
+          Saiba Mais
+        </Link>
+      </section>
+    )
+  }
+}
+
+FeaturedSection.propTypes = {
+  level: PropTypes.string,
```

Comment:
`level` must be `PropTypes.number` or even better: `PropTypes.oneOf([1, 2])`

Reply:
Done.